src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
+  } else if (days >= 3) {
+    result = result - 20;
+  }
```

Comment:
You can simplify the control flow by using `return` inside the `if` block and removing the `else` according to checklist items 4 and 5. This will make the code cleaner and easier to understand.